package.json

```diff
@@ -10,6 +10,7 @@
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run generate:readme:toc",
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "devTest": "ag test/asyncapi.yaml ./ -o output --force-write",
```

Comment:
please use `:` notation for scripts name to be consistent, in this case `dev` would be enough tbh